modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/js/ckeditor5/ClassicEditor.tsx

```diff
@@ -0,0 +1,69 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {CKEditor} from '@ckeditor/ckeditor5-react';
+import {
+	Bold,
+	ClassicEditor as BaseClassicEditor,
+	EditorConfig,
+	Essentials,
+	GeneralHtmlSupport,
+	Italic,
+	Paragraph,
+	Style,
+	StyleConfig,
+	Underline,
+} from 'ckeditor5';
+import React from 'react';
+
+import Timestamp from './plugins/Timestamp';
+
+interface ICKEditor5EditorConfig extends EditorConfig {
+	editorTransformerURLs?: Array<string>;
+}
+
+const ClassicEditor = ({config}: {config?: ICKEditor5EditorConfig}) => {
```

Comment:
We should be strict here and not allow `editor` ref leak, like in [existing editor](https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/js/editor/BaseEditor.js#L148). This, coupled with global `CKEDITOR` variable API, enabled product teams to make all sorts of weird and wonderful hacks, that caused a lot of problems in the end. If there is a need to expose some internal API, we can add an API layer here, and hopefully make sure it doesn't break anything. With CKE4, we had no control at all.